composer.json

```diff
@@ -14,7 +14,7 @@
         "illuminate/support": "^5.5,<5.8",
         "illuminate/console": "^5.5,<5.8",
         "illuminate/filesystem": "^5.5,<5.8",
-        "barryvdh/reflection-docblock": "^2.0.4",
+        "barryvdh/reflection-docblock": "dev-master#9b08619",
```

Comment:
This needs to become a new version of reflection-doclock

Reply:
Is tagged as 2.0.5

Reply:
Thanks, I updated it!